clients/src/main/java/org/apache/kafka/common/Cluster.java

```diff
@@ -327,6 +350,18 @@ public Node controller() {
         return controller;
     }
 
+    public Collection<Uuid> topicIds() {
+        return topicIds.values();
+    }
+
+    public Uuid getTopicId(String topic) {
+        return topicIds.getOrDefault(topic, Uuid.ZERO_UUID);
+    }
+
+    public String getTopicName(Uuid topiId) {
```

Comment:
nit: `Uuid topicId`

Reply:
Also, for getTopicId, we do getOrDefault. Is there a reason we wouldn't do that here?

Reply:
Here, I try to use getOrDefault but don't know what the default value is(it seems that "" is not a good default topicName). so just return null and let the caller decide, suggestions are welcomed if you have good ideas.

Reply:
I see...maybe I missed it, but I did not see the method used. What use case would you expect for topic names in Cluster?

Reply:
This is not used so far but will be used when we get TopicMetadata by topicId, so just add it in advance.